docs/design/latency_definitions/communication.md

```diff
@@ -0,0 +1,170 @@
+# Communication
+
+The communication latency is the time difference between the time the message is passed from the ownership of the callback to the next callback.
+
+$$
+l_{comm} = t_{receive} - t_{transfer}
+$$
+
+In particular, if transfer of ownership is publish and receive of ownership is subscribe, it is defined as follows
```

Comment:
As transmission is replaced by publish, but reception is subscription in the context of ROS 2, the expression is translated into the next one.